web-animations-1/Overview.bs

```diff
@@ -5743,9 +5762,9 @@ animation.play();</pre>
         <em>and</em>
     * Its [=replace state=] is <em>not</em> [=removed replace state|removed=].
 
-    For the purposes of this method
-    a [=pseudo-element=] is considered to be
-    a [=child=] of its [=originating element=].
+    Note: Since an {{Animation}} targeting a <a>pseudo-element</a> has the
+    <a>originating element</a> as its <a>target element</a>, calling {{Animatable/getAnimations}}
+    on an element will return <a>animations</a> targeting both the element itself and its pseudo-elements.
```

Comment:
It should only return them if `subtree: true` is set. I think the want to keep roughly the original spec text here.

Reply:
See my above comments about hanging Animations off of elements.

Reply:
One alternative proposed by @stephenmcgruer if to add a `pseudos` option to select whether just to get Animations targeting the element's base style or whether to get Animations targeting target pseudo-elements as well (i.e. all Animations hanging off the element).

Reply:
Let's keep the existing behavior for now. If you'd like to change it, let's use a separate issue / PR for that.

Reply:
Ok, it's a bit odd, but having a separate issue for that makes sense. I was trying to keep the invariant of it returns everything hanging off that element instead of the invariant of iit only returning the same set of animations as before.